www/index.html

```diff
@@ -1,20 +1,59 @@
 <html>
 <head>
-  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet" type="text/css">
-  <script src="https://apis.google.com/js/api:client.js"></script>
-  <script type="text/javascript" src="js/authenticate.js" ></script>
- <link rel="stylesheet" type="text/css" href="style/boilerplate.css">  </head>
-  <body>
-  <!-- In the callback, you would hide the gSignInWrapper element on a
-  successful sign in -->
-  <div id="gSignInWrapper">
-    <span class="label">Sign in with:</span>
-    <div id="customBtn" class="customGPlusSignIn">
-      <span class="icon"></span>
-      <span class="buttonText">Google</span>
-    </div>
-  </div>
-  <div id="name"></div>
-  <script>startApp();</script>
+	<link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet" type="text/css">
+	<script src="https://apis.google.com/js/api:client.js"></script>
+	<script type="text/javascript" src="js/authenticate.js" ></script>
+	<script type="text/javascript" src="js/chartAPI.js" ></script>
+	<script type="text/javascript" src="js/chart.js" ></script>
+
+	<link rel="stylesheet" type="text/css" href="style/boilerplate.css">  </head>
+<body>
+	<!-- In the callback, you would hide the gSignInWrapper element on a
+	successful sign in -->
+	<div id="gSignInWrapper">
+		<span class="label">Sign in with:</span>
+		<div id="customBtn" class="customGPlusSignIn">
+			<span class="icon"></span>
+			<span class="buttonText">Google</span>
+		</div>
+	</div>
+	<div id="name"></div>
+  
+	<script>startApp();</script>
+	
+	<canvas id="canvas" height="450" width="450"></canvas>
+	
+	<script>
```

Comment:
Move this into separate file, called  load_index.js.
This should be the entry point.